sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -4934,6 +4934,7 @@ public partial class GalleryArtifactVersionFullSource : Azure.ResourceManager.Co
         public string CommunityGalleryImageId { get { throw null; } set { } }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override System.Uri Uri { get { throw null; } set { } }
+        public string VirtualMachineId { get { throw null; } set { } }
```

Comment:
by its description, this is an arm resource id.
Could we change the format of this property to `arm-id`?

Reply:
You could do this by adding:
```
GalleryArtifactVersionFullSource.virtualMachineId: -|arm-id
```
to the `rename-mapping` list in `autorest.md`, then please regen the code and API file.